app/assets/javascripts/s3_direct_upload.js.coffee

```diff
@@ -63,7 +63,7 @@ $.fn.S3Uploader = (options) ->
         data.context.find('.bar').css('width', progress + '%')
 
     done: (e, data) ->
-      content = build_content_object $uploadForm, data.files[0], data.result
+         = build_content_object $uploadForm, data.files[0], data.result
```

Comment:
maybe i'm missing something here in coffee syntax, but is this right?

Reply:
Oh, this is my mistake, i already fix it